cypress-tests/cypress/e2e/admin/pageBuilder/category/pageCategoriesSearchAndSort.cy.ts

```diff
@@ -0,0 +1,153 @@
+context("Page Builder - Category Search&Sort", () => {
+    const categoryData1 = {
+        name: "ABC",
+        slug: "ABC",
+        layout: "static",
+        url: "/ABC/"
+    };
+    const categoryData2 = {
+        name: "EDF",
+        slug: "EDF",
+        layout: "static",
+        url: "/EDF/"
+    };
+    const categoryData3 = {
+        name: "GHI",
+        slug: "GHI",
+        layout: "static",
+        url: "/GHI/"
+    };
+    const categoryData4 = {
+        name: "!#$%&/()=?*",
+        slug: "!#$%&/()=?*",
+        layout: "static",
+        url: "/!#$%&/()=?*/"
+    };
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllCategories();
+        cy.wait(1500);
+        cy.pbCreateCategory(categoryData1);
+        cy.pbCreateCategory(categoryData2);
+        cy.pbCreateCategory(categoryData3);
+        cy.pbCreateCategory(categoryData4);
+    });
+
+    afterEach(() => {
+        cy.pbDeleteAllCategories();
+    });
+
+    it("Should be able to create, edit, and immediately delete a category", () => {
+        cy.visit("/page-builder/categories");
+        // Assert all of the programatically created categories are being properly displayed.
+        cy.contains(categoryData1.name).should("exist");
```

Comment:
Check if we can do these assertions in a container with a test ID, and not globally on the document level. The same way we did with template list search / cy.within.